pkg/permissions/config.go

```diff
@@ -8,11 +8,15 @@ import (
 
 // Config defines the permissions configuration structure
 type Config struct {
-	// URL is the URL checks should be executed against
+	// URL is the URL checks should be executed against.
+	// Considered unconfigured if empty.
 	URL string
 
 	// IgnoreNoResponders will ignore no responder errors when auth relationship requests are published.
 	IgnoreNoResponders bool
+
+	// AllowUnconfiguredChecks permits permission checks when unconfigured.
+	AllowUnconfiguredChecks bool
```

Comment:
This strikes me as a bit of an odd name. Maybe something like `DefaultAllow` or `DefaultAllowAll` would be better; the distinction between "configured" and "unconfigured" is confusing.

Reply:
I too am not a fan of the name, however I wanted to be clear that this only takes affect when the url is not defined and therefore we're not processing standard checks.

If you still think one of these is a more appropriate name, I can update it to that.

Reply:
Maybe the suggested doc comment helps?

Reply:
indeed it does, thanks.